backend/controllers/bulkAppointmentsUpload.js

```diff
@@ -60,7 +60,7 @@ const bulkAppointmentsUploadFactory = (csvParserService, offenderLoader, logErro
           const offenderNosNotFound = getNonExistingOffenderNumbers(offendersFromCsv, prisonerList)
 
           if (offenderNosNotFound.length === fileContent.length) {
-            res.redirect('/bulk-appointments/no-appointments-added')
+            res.redirect('/bulk-appointments/no-appointments-added?reason=offendersNotFound')
             res.end()
```

Comment:
What's the res.end() used for?

Reply:
No idea, I'm just changing it now to return res.redirect...